<ui version="4.0" >
 <author>Aaron Racicot</author>
 <class>NextPolygon</class>
 <widget class="QDialog" name="NextPolygon" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OpenOceanMap - Next Polygon</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="font" >
      <font>
       <family>Tahoma</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="title" >
      <string/>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <property name="geometry" >
       <rect>
        <x>12</x>
        <y>3</y>
        <width>461</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <family>Tahoma</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_1" >
           <property name="minimumSize" >
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font" >
            <font>
             <family>Tahoma</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text" >
            <string>Pennies:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_1" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="minimumSize" >
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font" >
            <font>
             <family>Tahoma</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="pl_label" >
           <property name="minimumSize" >
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font" >
            <font>
             <family>Tahoma</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer" >
           <property name="font" >
            <font>
             <family>Tahoma</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="habitat_combo_label" >
           <property name="minimumSize" >
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font" >
            <font>
             <family>Tahoma</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text" >
            <string>Habitat Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="habitat_combo" >
           <property name="minimumSize" >
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font" >
            <font>
             <family>Tahoma</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <item>
            <property name="text" >
             <string>Unknown</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Hard Bottom</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Soft Sandy</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Mixed</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2" >
           <property name="font" >
            <font>
             <family>Tahoma</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" native="1" name="widget" >
         <property name="font" >
          <font>
           <family>Tahoma</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pbnShapeFinished" >
            <property name="font" >
             <font>
              <family>Tahoma</family>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text" >
             <string>Finished With Fishery</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbnShapeDiscard" >
            <property name="font" >
             <font>
              <family>Tahoma</family>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text" >
             <string>Discard Last Shape</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbnMoreShapes" >
            <property name="font" >
             <font>
              <family>Tahoma</family>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text" >
             <string>More Shapes This Fishery</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
